bootstrap/src/main/java/org/wildfly/swarm/bootstrap/env/ApplicationEnvironment.java

```diff
@@ -271,6 +271,24 @@ public ClassLoader getBootstrapClassLoader() throws ModuleLoadException {
         return this.manifests;
     }
 
+    public NativeDeploymentFactory nativeDeploymentFactory() {
+        return this.nativeDeploymentFactory.updateAndGet( (factory)->{
+            if ( factory != null ) {
+                return factory;
+            }
+
+            try {
+                Module module = Module.getBootModuleLoader().loadModule(ModuleIdentifier.create("org.wildfly.swarm.container", "runtime"));
+                Class<NativeDeploymentFactory> cls = (Class<NativeDeploymentFactory>) module.getClassLoader().loadClass("org.wildfly.swarm.container.runtime.NativeDeploymentFactoryImpl");
+                return cls.newInstance();
+            } catch (ModuleLoadException | ClassNotFoundException | InstantiationException | IllegalAccessException e) {
+                //e.printStackTrace();
```

Comment:
so this is bootstrap, so prior to logging configuration mostly and it is an ignorable error, so just removing that.